core/lib/kv/keys.ts

```diff
@@ -1,2 +1,3 @@
 export const STORE_STATUS_KEY = 'storeStatus';
-export const routeCacheKvKey = (pathname: string) => `v3_${pathname}`;
+export const kvKey = (key: string, channelId?: string) =>
+  `${channelId ?? process.env.BIGCOMMERCE_CHANNEL_ID ?? '1'}_${key}`;
```

Comment:
Thoughts on moving the `KV` namespace logic to this fn? .

Reply:
Will do